src/mediaelement.js

```diff
@@ -86,6 +86,16 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
         this.playbackRate = value || 1;
         this.media.playbackRate = this.playbackRate;
     },
+    
+    /**
+     * Get the audio source playback rate.
+     */
+    getPlaybackRate: function () {
+    	if (this.media.playbackRate == null) {
+    		this.media.playbackRate = 1;
+    	}
+        return this.media.playbackRate;
```

Comment:
I'd rather not set a value in the getter. How about just `return this.media.playbackRate || 1`?